BitwardenShared/UI/Vault/VaultItem/CipherItemState.swift

```diff
@@ -136,25 +136,27 @@ struct CipherItemState: Equatable {
 
     /// Whether or not this item can be deleted by the user.
     var canBeDeleted: Bool {
-        // New permission model from PM-18091
-        if restrictCipherItemDeletionFlagEnabled, let cipherPermissions = cipher.permissions {
-            return cipherPermissions.delete
+        // backwards compatibility for old server versions
+        guard restrictCipherItemDeletionFlagEnabled, let cipherPermissions = cipher.permissions else {
+            guard !collectionIds.isEmpty else { return true }
+            return collections.contains { collection in
+                guard let id = collection.id else { return false }
+                return collection.manage && collectionIds.contains(id)
+            }
         }
 
-        guard !collectionIds.isEmpty else { return true }
-        return collections.contains { collection in
-            guard let id = collection.id else { return false }
-            return collection.manage && collectionIds.contains(id)
-        }
+        // New permission model from PM-18091
+        return cipherPermissions.delete
     }
```

Comment:
🤔 Maybe I'm missing something but as far as I see the behavior is exactly the same as before. Why is this change needed?